package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "lint": "./node_modules/.bin/eslint -c .eslintrc webpack/",
+    "lint:spelling": "eslint ./",
```

Comment:
Since in this branch we still use `eslint`, `npm run lint:spelling` is effectively the same as `npm run lint` (except it might use global `eslint` command, I guess).

This also means `.github/workflows/js_tests.yml` should be back to `run: npm run lint`, there's single command doing both things.

Reply:
Awesome thanks !